pkg/coscheduling/coscheduling.go

```diff
@@ -29,23 +29,34 @@ import (
 	corelisters "k8s.io/client-go/listers/core/v1"
 	"k8s.io/klog"
 	"k8s.io/kubernetes/pkg/api/v1/pod"
+	podapi "k8s.io/kubernetes/pkg/api/v1/pod"
```

Comment:
Small Tips. I find "k8s.io/kubernetes/pkg/api/v1/pod" is usually named  `podutil` in kubernetes.